<?xml version="1.0"?>
<launch>

  <arg name="debug" default="false"/>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find map_exploration)/custom_config.conf" if="$(arg debug)"/>

  <node pkg="map_exploration" type="explore" respawn="false" name="explore" output="screen">
    <param name="robot_base_frame" value="base_link"/>
    <param name="costmap_topic" value="/map"/>
    <param name="costmap_updates_topic" value="/map_updates"/>
    <param name="visualize" value="true"/>
    <param name="planner_frequency" value="0.5"/>
    <param name="potential_scale" value="3.0"/>
    <param name="orientation_scale" value="0.0"/>
    <param name="gain_scale" value="1.0"/>
    <param name="min_frontier_size" value="0.35"/>
    <param name="tf_timeout" value="2.0"/>
    <param name="retries" value="10"/>
  </node>

</launch>